File: src/main/java/io/github/hydrazinemc/oxidiser/OxidiserPreLaunchInitializer.java
package io.github.hydrazinemc.oxidiser;

import com.llamalad7.mixinextras.MixinExtrasBootstrap;

import net.fabricmc.loader.api.entrypoint.PreLaunchEntrypoint;

public class OxidiserPreLaunchInitializer implements PreLaunchEntrypoint {
    @Override
    public void onPreLaunch() {
        MixinExtrasBootstrap.init();
    }
}

File: src/main/kotlin/io/github/hydrazinemc/oxidiser/OxidiserPreLaunchInitializer.kt
package io.github.hydrazinemc.oxidiser

import com.llamalad7.mixinextras.MixinExtrasBootstrap
import org.quiltmc.loader.api.ModContainer
import org.quiltmc.loader.api.entrypoint.PreLaunchEntrypoint

class OxidiserPreLaunchInitializer : PreLaunchEntrypoint {
	override fun onPreLaunch(mod: ModContainer?) {
		MixinExtrasBootstrap.init()
	}
}